app/gateway/2.6.x/install-and-run/upgrade.md

```diff
@@ -25,13 +25,15 @@ instructions on [database migration](#migrate-db), especially for Cassandra user
 - Upgrading from from 1.x is a major upgrade. Follow the [Version Prerequisites](#version-prerequisites).
 Be aware of any noted breaking changes as documented in the version to which you are upgrading.
 
-### Version Prerequisites
+### Version prerequisites
 
 If you are not on Kong Gateway 2.5.x, you must first incrementally
 upgrade to 2.5.x before upgrading to 2.6.x. Zero downtime is possible but _not_
 guaranteed if you are upgrading incrementally between versions, from 0.36.x to 1.3.x to
 1.5.x to 2.1.x to 2.2.x to 2.3.x to 2.4.x to 2.5.x to 2.6.x., plan accordingly.
 
+#### Enterprise instructions
```

Comment:
```suggestion
```
Honestly, I'd argue that we don't actually need to include this section at all, _or_ remove links. 

Alternatively, to avoid removing content (if that's a concern), we could link over to the `/gateway-oss/` parallels of these topics - these instructions aren't enterprise-specific, this section just doesn't exist in the OSS docs because they didn't think this level of detail was necessary.